src/app/core/internal/impl/CircuitInternal.ts

```diff
@@ -282,9 +282,7 @@ export class CircuitInternal extends Observable {
     }
 
 
-    //
-    // Convenience functions around CircuitOps
-    //
+          this.objStorage.set(op.c.id, op.c);
```

Comment:
it's not compiling because this statement is floating in the middle of nowhere haha